### PR TITLE
fix(plugin): respect `--insecure`

### PR DESCRIPTION
**Pull request by knqyf263**
## Description
`--insecure` is not respected for plugin subcommands now.

## Related issues
- Close https://github.com/aquasecurity/trivy/issues/7021

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

**Comment by knqyf263** [comment]
I hope the tests will pass 🤞  I want a review queue option that marks PR ready for review after all tests pass, similar to a merge queue.

File: pkg/commands/app.go
Comment (DmitriyLewen) [inline pkg/commands/app.go:1]: IIUC we need to add `insecure` option for `run` flag.

Comment (knqyf263) [reply pkg/commands/app.go:1]: Thanks. Added https://github.com/aquasecurity/trivy/pull/7022/commits/7b1eda63cd4979aaf2fc92a2da44193680f28f9f

**Review by DmitriyLewen — APPROVED** [review]
LGTM

**Pull request merged**